src/main/java/mods/railcraft/common/blocks/logic/InventoryLogic.java

```diff
@@ -41,7 +41,7 @@ protected void dropItem(ItemStack stack) {
 
     @Override
     public boolean isUsableByPlayer(EntityPlayer player) {
-        return adapter.isUsableByPlayer(player);
+        return true; //adapter.isUsableByPlayer(player); will cause infinite loop
```

Comment:
How? This is kind of necessary or you can open GUIs from across the map.

Reply:
But apparently the standalone inv delegates this method call back to its owner, so the current implementation would cause a stack overflow error.

Reply:
But the adapter points at a helper function in TileRailcraft and its own implementation for carts, neither of which refer to the standalone inv.

Reply:
Didn't notice that fact. My bad, fixed.

Reply:
Are you confusing the tile/cart adapter with the inventory object?